src/pages/accessibility/index.mdx

```diff
@@ -66,7 +66,7 @@ Applicable Standards/Guidelines: This report covers the degree of conformance fo
 | [2.5.2 Pointer Cancellation](https://www.w3.org/TR/WCAG21/#pointer-cancellation) (Level A 2.1 only)                         | Supports           | PrairieLearn does not complete actions on the down-event of the pointer.                                                                                                   |
 | [2.5.3 Label in Name](https://www.w3.org/TR/WCAG21/#label-in-name) (Level A 2.1 only)                                       | Supports           | The text and accessible name of inputs match. User interface elements that use icons or symbols have appropriate accessible labels.                                        |
 | [2.5.4 Motion Actuation](https://www.w3.org/TR/WCAG21/#motion-actuation) (Level A 2.1 only)                                 | Supports           | PrairieLearn does not rely on device motion or user motion.                                                                                                                |
-| [3.1.1 Language of Page](http://www.w3.org/TR/WCAG20/#meaning-doc-lang-id) (Level A)                                        | Partially Supports | PrairieLearn does not contain multi-language content by default. There is no way for users to override the language on user-generated content.                             |
+| [3.1.1 Language of Page](http://www.w3.org/TR/WCAG20/#meaning-doc-lang-id) (Level A)                                        | Supports           | PrairieLearn's own UI uses English, which is declared in a `lang` attribute. User-generated content may be written in languages other than English.                        |
```

Comment:
Note that user-generated content can use the `lang` attribute on specific HTML elements in a question if needed.